src/types/onyx/Transaction.ts

```diff
@@ -81,7 +81,7 @@ type Comment = {
     splits?: Split[];
 
     /** Violations that were dismissed */
-    dismissedViolations?: Record<ViolationName, Record<string, string>>;
+    dismissedViolations?: Record<ViolationName, Record<string, string | number>>;
```

Comment:
Not a blocker just as curiosity: I think oldDot is storing a string here and newDot will store a number due to using different endpoints to dismiss violations 😅